plopfile.js

```diff
@@ -0,0 +1,175 @@
+module.exports = function (plop) {
```

Comment:
Plop docs are using ES for their examples, can we use it here too?

Reply:
done